examples/hosts/app-integration/external-data/src/model-interface/index.ts

```diff
@@ -32,6 +32,17 @@ export interface IAppModel extends IEventProvider<IAppModelEvents> {
 	 */
 	readonly baseDocument: IBaseDocument;
 
+	/**
+	 * Sets the current leader of the document to the clientID that claimed leadership.
+	 */
+	readonly handleClaimLeadership: () => void;
+
+	/**
+	 *
+	 * @returns string with current user's clientID.
```

Comment:
What does it mean for this to return `undefined`?

Reply:
From what I understand, when a client first connects to the container, it doesn't immediately have a clientId; It only receives one after a successfully connection with the server.

So undefined means the client hasn't yet finished connecting or there was some error in the connection process.